app/connectors/EmailClient.scala

```diff
@@ -178,11 +163,28 @@ trait CSREmailClient extends OnlineTestEmailClient with AssessmentCentreEmailCli
 
 }
 
-trait OnlineTestEmailClient {
+sealed trait OnlineTestEmailClient {
+
+  self: EmailClient =>
+
   def sendOnlineTestInvitation(to: String, name: String, expireDateTime: DateTime)(implicit hc: HeaderCarrier): Future[Unit]
   def sendOnlineTestExpired(to: String, name: String)(implicit hc: HeaderCarrier): Future[Unit]
-  def sendTestExpiringReminder(to: String, name: String, timeLeft: Int,
-    timeUnit: TimeUnit, expiryDate: DateTime)(implicit hc: HeaderCarrier): Future[Unit]
+  def sendTestExpiringReminder(to: String, name: String, timeLeftInHours: Int,
+                               timeUnit: TimeUnit, expiryDate: DateTime)(implicit hc: HeaderCarrier): Future[Unit]
+
+  protected def sendExpiringReminder(template: String)(to: String, name: String, timeLeftInHours: Int,
```

Comment:
The advantage here that the phase1/pase2 client vill populate the template value so the caller won't need to bother about it.